src/terra-dev-site/guide/GeneralGuidelines.l.guide.mdx

```diff
@@ -0,0 +1,8 @@
+# General Guidelines
+
+Keep in mind these things when developing terra components
+
+* Terra components are black boxes. You should not rely on internal implementation specifics. This includes DOM structure, classes, and undocumented side effects.
```

Comment:
Should we mention that css files are internal implementation and not considered part of the api?